target/scripts/check-for-changes.sh

```diff
@@ -22,17 +22,6 @@ _log_with_date 'debug' 'Starting changedetector'
 #            to be properly set.
 _obtain_hostname_and_domainname
 
-if ! cd /tmp/docker-mailserver &>/dev/null
-then
-  _exit_with_error "Could not change into '/tmp/docker-mailserver/' directory" 0
-fi
```

Comment:
Where is the `cd` done instead? I was at first not really thinking about it, but now, I cannot see where the directory change is happening instead :D

Reply:
I don't know either, but maybe there is some logic that we don't currently know of that depends on being in the correct directory...

Reply:
It's not performed as it shouldn't need to be anymore due to everything using absolute paths?

I oddly cannot reproduce the failure on my local VM instance for the branch either. It might be something with the environment? But it's not clear why the changes would be breaking that particular test?

I could revert changes in the PR until the CI tests pass I guess to find out what it's unhappy about?